airbyte-cdk/.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "workbench.colorCustomizations": {
+        "activityBar.background": "#1E128E",
+        "titleBar.activeBackground": "#2A19C7",
+        "titleBar.activeForeground": "#FDFDFF"
+    },
+    "python.formatting.provider": "black"
+}
```

Comment:
Please remove this config file from PR. It better to use pre-commit since it using same config as it used by CI on formating stage. You can skip mypy checking  by running ti with env SKIP:
```bash
SKIP=mypy git commit ....
```
ot just setting alias:
```bash
alias git="SKIP=mypy git"
```